tests/wpt/mozilla/tests/css/perspective.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>CSS Test: parsing perspective property</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+</head>
+
+<body>
+    <div id="perspective0px" style="perspective: 0px"></div>
```

Comment:
instead of having four elements, we could just have:

`<div id="test"></div>`

and make the test like follows:

```js
test(function() {
    var test = document.getElementById('test');
    var tests = [
        ["0px", "0px"],
        ["100px", "100px"],
        ["-100px", ""],
        ["200PX", "200px"],
    ];
    for (var i = 0; i < tests.length; ++i) {
        var specified = tests[i][0];
        var expected = tests[i][1];
        test.style.perspective = specified;
        assert_equals(test.style.perspective, expected);
    }
}, "Parsing of the perspective property");
```